kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -56,6 +56,48 @@ class RobotState;
 namespace kinematics
 {
 
+/*
+ * @enum DiscretizationMethods
+ *
+ * @brief Flags for choosing the type discretization method applied on the redundant joints during an ik query
+ */
+namespace DiscretizationMethods
```

Comment:
Brw, another option is to include a prefix in the enum values themselves, which is what we've done more often in MoveIt.